utils/kata-manager.sh

```diff
@@ -584,6 +594,46 @@ install_nerdctl()
 		done
 
 	info "$project installed\n"
+
+	sudo mkdir -p /opt/cni/bin
```

Comment:
I am not sure this step is required. As at this step, the path of `/opt/cni/bin` does actually exist, right ?

Reply:
This path needs to be created for a fresh install.

Reply:
I find that there's two different place doing the same operation" mkdir -p /opt/cni/bin", why ?

Reply:
@Apokleos We do this separately for containerd install and nerdctl install.
When nerdctl is being installed, the nerdctl tarball includes containerd and cni files as well, we skip install_containerd in that case.
While containerd files are installed in correct location, the cni files needs to be installed under /opt/cni/bin which is what is happening here.

When only containerd is installed instead of nerdctl, the containerd tarball does not contain cni plugin files. I explicitly install cni in that case which is why you see ` mkdir -p /opt/cni/bin` operation there.

As you can see, the two cases are exclusive.